egs2/bur_openslr80/asr1/local/data_prep.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
```

Comment:
Just a note, if it is identical to the original code, please use symlink instead of copy it

Reply:
Hey, no there are changes. The data prep file is specific to the Burmese OpenSLR data format.